packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts

```diff
@@ -322,9 +338,9 @@ export default class TableResize implements EditorPlugin {
         }, ChangeSource.Format);
     };
 
-    private setCurrentTable(table: HTMLTableElement, rect: Rect): void;
+    private setCurrentTable(table: HTMLTableElement): void;
     private setCurrentTable(table: null): void;
-    private setCurrentTable(table: HTMLTableElement, rect?: Rect) {
+    private setCurrentTable(table: HTMLTableElement) {
```

Comment:
We can remove the function override now.

Reply:
done